webpack/assets/javascripts/foreman_navigation.js

```diff
@@ -20,3 +20,15 @@ export const changeOrganization = org => {
 export const changeActive = active => {
   store.dispatch(LayoutActions.changeActiveMenu({ title: active }));
 };
+
+export function showContent(layout, unsubscribe) {
+  const content = () => {
+    $('#content').show();
+    unsubscribe();
+  };
+  // workaround for pages with no layout object
+  if ($('#layout').length === 0) content();
+  else if (layout.items.length && !layout.isLoading) {
```

Comment:
nitpick: does it make sense to swap the query, as I assume going over the layout obj is faster doing a dom query (and its the more common usage case)?

Reply:
makes sense :+1: